main-app/MoveMaster-app/Assets/Scripts/Scoreboard/ScoreboardManager.cs

```diff
@@ -0,0 +1,43 @@
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+
+public class ScoreboardManager : MonoBehaviour
+{
+    public static ScoreboardManager Instance;
+    [SerializeField]
+    private TMP_Text scorePoints;
+    [SerializeField]
+    private int score=0;
+
+    void Awake(){
+        if (Instance != null && Instance != this) 
+    { 
+        Destroy(this); 
+    } 
+    else 
+    { 
+        Instance = this; 
+    } 
+    }
+
+    void Start()
+    {
+        setScore(0);
+    }
+
+    public void setScore(int newScore){
+        score = newScore;
+        scorePoints.SetText(newScore.ToString());
+    }
+
+    public int getScore(){
```

Comment:
Ponownie delegat + PascalCase

Reply:
Można bez delegat. Jak ktoś lubi to wciąż może używać

Reply:
Nie do poprawy. Chyba że chcesz sobie to ogarnąć